src/transformers/training_args.py

```diff
@@ -481,6 +482,11 @@ class TrainingArguments:
             are also available. See the [Ray documentation](
             https://docs.ray.io/en/latest/tune/api_docs/analysis.html#ray.tune.ExperimentAnalysis.get_best_trial) for
             more options.
+        ddp_timeout (`int`, *optional*, defaults to `1800`):
```

Comment:
```suggestion
        ddp_timeout (`int`, *optional*, defaults to 1800):
```
No code blocks for ints :-)